docs/development/release.rst

```diff
@@ -6,74 +6,58 @@
 How to make a Gammapy release
 *****************************
 
-This page contains step-by-step instructions how to make a Gammapy release.
+This page contains step-by-step instructions how to make a Gammapy release. We mostly follow the
+`Astropy release instructions <https://docs.astropy.org/en/latest/development/releasing.html>`__
+and just lists the additional required steps.
 
-Overview
---------
 
-We have structured the procedure in three phases:
+Feature Freeze and Branching
+----------------------------
 
-#. "Pre release" -- on a day (or several days) before making the release
-#. "Make release" -- on the day of making the release
-   (tag the stable version and make source release on PyPI)
-#. "Post release" -- on the day when announcing the release
-   (ideally only two or three days after making the release)
+#. Follow the `Astropy feature freeze and branching instructions <https://docs.astropy.org/en/latest/development/releasing.html#start-of-a-new-release-cycle-feature-freeze-and-branching>`__
+   Instead of updating the `whatsnew/<version>.rst` update the `docs/release-notes/<version>.rst`.
 
-The purpose of writing the procedure down explicitly is to make it easy for
-anyone to make a release (as opposed to one or a few people with secret
-knowledge how to do it). It's also good to not have to remember this stuff, and
-to avoid errors or forgotten steps.
 
-Note that we currently don't do bugfix releases, i.e. making a release is always simple in the sense
-that you just need to create a branch off of ``master``.
-Making bugfix releases would be more difficult and involve identifying commits with bug fixes and
-backporting those to the stable branches.
+Releasing the first major release candidate
+-------------------------------------------
 
-In these notes we'll use the Gammapy 0.14 release as an example.
+#. Add an entry for the release candidate like `v1.0rc1` or `v1.1rc1` in the ``download/index.json`` file in the `gammapy-web repo <https://github.com/gammapy/gammapy-webpage>`__.
+   The ``datasets`` entry should point to dev version of `gammapy-data`. This allows to fix bugs in the data during the release candidate testing.
 
-Pre release
------------
+#. Fill the changelog ``docs/release-notes/<version>.rst`` for the version you are about to release.
+#. Update the ``CITATION.cff`` by running the ``dev/prepare-release.py`` script.
+#. Open a PR including both changes and mark it with the `backport-v<version>.x` label.
+   Gather feedback from the Gammapy user and dev community and finally merge and backport to the `v<version>.x` branch.
```

Comment:
This should be after the rc is released no? Why is user feedback for here?

Reply:
Yes, at this stage it could be omitted, but it might still be reasonable to have some time for other people to look at the proposed changelog, because we do not auto-generate it. So the content is flexible and can be discussed...

Reply:
The same goes for the changes to `CITATION.cff` in case contributors would still like to join the auto-generated author list. I think the review should happen before the first release candidate, because otherwise we will always do `rc2`?